docs/apispec-docs/src/main/scala/sttp/tapir/docs/apispec/schema/TSchemaToASchema.scala

```diff
@@ -33,6 +33,12 @@ private[docs] class TSchemaToASchema(
           case TSchemaType.SNumber()  => ASchema(SchemaType.Number)
           case TSchemaType.SBoolean() => ASchema(SchemaType.Boolean)
           case TSchemaType.SString()  => ASchema(SchemaType.String)
+          case TSchemaType.SProduct(fields) if schema.attribute(TSchema.ProductAsArray.Attribute).map(_.productAsArray).getOrElse(false) =>
+            // TODO: Draft04 uses alternative form of `items` instead
```

Comment:
I think Draft04 is only used here: https://github.com/softwaremill/tapir/blob/17af1369bb89a7f68f9a0125cce41b1d105c90f3/docs/apispec-docs/src/main/scala/sttp/tapir/docs/apispec/schema/TapirSchemaToJsonSchema.scala#L12

So maybe let's change it to 2020-12? That's what used by apispec

Reply:
I think already output of TapirSchemaToJsonSchema looks more like 2020-12 than Draft04. Few tests show schemas validate correctly (against meta schema). Can I push this change to this PR?

Reply:
yes please!